### PR TITLE
Add support for scroll testnet

### PR DESCRIPTION
**Pull request by bhargavaparoksham**
Fixes: #2777 

## Description

Add support for scroll testnet

## Checklist

This PR:

- [x] Includes only changes relevant to the original ticket. Significant refactoring needs to be separated.
- [x] Doesn't disable eslint rules.
- [x] Doesn't work around the type checker (including but not limited to: type casts, non-null assertions, `@ts-ignore`, unjustified optional values).
- [x] Doesn't contain commented out code.
- [x] If adding/updating a feature, it adds/updates its test script on Notion.

### CODE REVIEW

File: packages/common/src/chains.ts [hunk 1]
```diff
@@ -99,6 +101,19 @@ export const zkSyncEraTestnet: Chain = {
   },
 };
 
+export const scrollTestnet: Chain = {
```
Comment (vacekj) [inline packages/common/src/chains.ts:104]: scroll sepolia should be available as a chain from viem/chains here.

File: packages/grant-explorer/src/features/api/utils.ts [hunk 2]
```diff
@@ -536,6 +542,29 @@ const FUJI_TOKENS: VotingToken[] = [
   },
 ];
 
+const SCROLL_TESTNET_TOKENS: VotingToken[] = [
+  {
+    name: "MTK",
+    chainId: ChainId.SCROLL_TESTNET,
+    address: "0xc2332031de487f430fae3290c05465d907785eda",
+    decimal: 18,
+    logo: TokenNamesAndLogos["DAI"],
```
Comment (vacekj) [inline packages/grant-explorer/src/features/api/utils.ts:551]: if we have a test logo let's use that instead of DAI to not mislead the user

**Comment by michellema1208** [comment]
hey @bhargavaparoksham - running into a few issues:

- [ ] I'm not able to apply to a round on Scroll, and it looks like it's due to a LIT error: 

Screen Shot 2024-01-10 at 9 44 17 AM

- [ ] The Builder "My Projects" page generally takes a really long time to load, and also seems to be missing Scroll Sepolia projects  

Screen Shot 2024-01-10 at 9 45 55 AM

**Review by vacekj — CHANGES_REQUESTED** [review]
scrollTestnet is ambiguous as a chain name, since there was previously a scroll alpha testnet, and there will be new testnets in the future. let's call it `scrollSepolia` to disambiguate.

**Comment by gravityblast** [comment]
closing for now since LIT doesn't support scroll testnet

**Pull request closed without merging**